charts/mattermost-enterprise-edition/values.yaml

```
@@ -97,7 +97,7 @@ mattermostApp:
   replicaCount: 2
   image:
     repository: mattermost/mattermost-enterprise-edition
-    tag: 5.27.0
+    tag: 5.28.0
```

please bump, was released yesterday
```suggestion
    tag: 5.28.1
```

---

Done also changed in the Chart.yaml as well